bottlerocket/types/src/agent_config.rs

```diff
@@ -31,7 +31,10 @@ pub enum SonobuoyMode {
     /// This is the default mode and will run all the tests in the e2e plugin which are marked
     /// `Conformance` which are known to not be disruptive to other workloads in your cluster.
     NonDisruptiveConformance,
-    //// This mode runs all of the Conformance tests.
+    /// An unofficial mode of running the e2e tests which removes some of the longest running tests
+    /// so that tests can complete in the fastest time possible while maximizing coverage.
+    ConformanceLite,
```

Comment:
If it is unofficial, why are we adding it? Will it be official at some point?

Reply:
"Unofficial" as in upstream K8s doesn't define what `conformance-lite` means. `non-disruptive-conformance` is also "unofficial" in the same sense. But it is a valid testing mode in sonobuoy that lets us do quick verification.